apps/juicebox_web/assets/js/videos/video.js

```diff
@@ -40,6 +41,8 @@ class Video extends Component {
   }
 
   render() {
+    console.log(this.props);
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)